Doc/Tutorial/chapter_testing.tex

```diff
@@ -311,7 +311,7 @@ \subsection{Writing a unittest-based test}
     unittest.main(testRunner=runner)
 \end{verbatim}
 
-In the division tests, we use \verb|assertAlmostEqual| instead of \verb|assertEqual| to avoid tests failing due to roundoff errors; see the \verb|unittest| chapter in the Python documentation for details and for other functionality available in \verb|unittest| (\href{http://docs.python.org/library/unittest.html}{online reference}).
+In the division tests, we use \verb|assertAlmostEqual| instead of \verb|assertEqual| to avoid tests failing due to roundoff errors; see the \verb|unittest| chapter in the Python documentation for details and for other functionality available in \verb|unittest| (\href{https://docs.python.org/2/library/unittest.html}{online reference}).
```

Comment:
--> https://docs.python.org/3/library/unittest.html

Reply:
I use the reference to Python 3 documentation.